src/utils/address.ts

```diff
@@ -76,4 +78,21 @@ export async function deriveAddressP2SH(index: number, storage: IStorage): Promi
     throw new Error('No multisig data');
   }
   return deriveAddressFromDataP2SH(multisigData, index, storage.config.getNetwork().name);
-}
\ No newline at end of file
+}
+
+/**
+ * Parse the public key and return an address.
+ *
+ * @param pubkey Hex string conveying the public key.
+ * @param network Address's network.
+ * @returns The address object from parsed publicKey or null if publicKey is nulish.
+ */
+export function getAddressFromPubkey(pubkey: string, network: Network): Address|null {
+  if (pubkey == null) {
+    return pubkey;
+  }
+
```

Comment:
```suggestion
export function getAddressFromPubkey(pubkey: string, network: Network): Address {
```

Since the pubkey is supposed to be a string we can be sure that it will not be null.
If it is, the caller is passing the wrong argument and we can expect an error.

Reply:
Done at https://github.com/HathorNetwork/hathor-wallet-lib/pull/613/commits/99b5968fe84fa4e8175111fc1026eb22b88ff413